mtools/test/test_mloginfo.py

```diff
@@ -400,6 +400,109 @@ def _test_rsinfo(self, logfile_path, **expected):
             print("results[%s] == %s" % (key, value))
             assert results.get(key) == value
 
+    def test_sharding_output(self):
+        self.tool.run('--sharding --errors --migrations %s' % self.logfile_path)
+        output = sys.stdout.getvalue()
+        lines = output.splitlines()
+        assert any(map(lambda line: 'SHARDING' in line, lines))
+
+        assert any(map(lambda line: 'Overview:' in line, lines))
```

Comment:
Test should generally be validating that the expected analysis has been performed. You could use the section headings as an anchor to find the expected details, but the important part of the test would be making sure that data like error messages and number of chunks moved from/to matches expectations.

Reply:
I decided to remove this test and add more detail to others so that the table expectations were tested more